frame/auction.ts

```diff
@@ -23,11 +23,20 @@ import { FetchJsonStatus, tryFetchJson } from "./fetch";
  * used has no use for scoring signals, but when more algorithms are supported,
  * this data will be passed to them.
  *
+ * The request's `keys` query parameter consists of all the ads' rendering URLs,
+ * escaped and then joined with unescaped commas, ordered first alphabetically
+ * by interest group name and then by the order given in the
+ * `joinAdInterestGroup` call, with only the first kept in case of duplicates.
+ * This leaks some information about interest groups and when they were joined
+ * to the trusted server, but that's okay, since it's trusted.
```

Comment:
Why not just sort by renderingUrl?  Then we leak nothing.

(if there's some reason this is impractical, that's probably OK, but the sort seems pretty easy?)

Reply:
Filed #104.